maap/Secrets.py

```diff
@@ -0,0 +1,129 @@
+import requests
+import logging
+import json
+from maap.utils import endpoints
+from maap.utils import requests_utils
+
+
+class Secrets:
+    """
+    Functions used for Member secrets API interfacing
+    """
+    def __init__(self, config, api_header):
+        self._api_header = api_header
+        # self._members_endpoint = members_endpoint
+        self._members_endpoint = "https://api.dit.maap-project.org/api/members/self"
```

Comment:
Assuming this is a test artifact. Just commenting to remind us to remove this.

Reply:
Yes, this was just for testing as I thought I would have to update the MAAP API config.